### ISSUE TITLE
When generating ethereum addresses, the results are different than what can be found on coinomi's version of this tool

### ISSUE THREAD

**Issue by thinkloop** [iancoleman/bip39#128]
When using the same mnemonic on coinomi's tool: https://coinomi.com/recovery-phrase-tool.html -
 the results differ for ethereum (other coins seem to match). The BIP32 Root Key is different. Which is the more correct implementation and why the difference?

Also what is the relationship between http://iancoleman.io/bip39, http://coinomi.com/recovery-phrase-tool.html and http://particl.github.io/bip39/bip39-standalone.html?

**Comment by iancoleman** [comment]
Coinomi uses these values for [their ethereum bip32 network values](https://github.com/Coinomi/bip39/commit/c04070d55a1baee3763db7594702261a0fc4ad5a#diff-7bde6632c0ae01de621a96709970ee73R680)
```
bip32: {
    public: 0xffffffff,
    private: 0xffffffff
},
```

This tool uses [the bitcoin network parameters](https://github.com/iancoleman/bip39/blob/b57b721e2cec18e0df07f1e35e1e72852f51bec4/src/js/index.js#L1364) when ethereum is selected
```
bip32: {
  public: 0x0488b21e,
  private: 0x0488ade4
},
```

Particl also [uses the bitcoin network parameters](https://github.com/particl/bip39/blob/15d8c6a5f3bdcf0e8457e36732dbb3196914c5d7/src/js/index.js#L1112)

Who's right? I don't know. Anyone with a link to the ethereum parameters for bip32 please post it.

The relationship between coinomi is they forked this repo and are currently "43 commits ahead, 290 commits behind iancoleman:master"

The relationship between particl.github.io is also forked from this repo and is currently "4 commits ahead, 103 commits behind iancoleman:master."

**Comment by thinkloop** [comment]
Thank you, well analyzed. I verified the implementations at ethtools.com and myetherwallet.com, and they both match yours. That's likely part of the reason coinomi needed to fork.

**Comment by avibrazil** [comment]
Wow, this is a serious break of “standards that aren’t standardized”.

If you use BIP32 tab and play with derivation path and hardenet flag you can match Coinomi addresses.

**Comment by avibrazil** [comment]
@thinkloop, what do you mean by saying you verified myetherewallet.com? Its not HD so it can't be compared to this tool under the BIP32 discussion.

**Comment by thinkloop** [comment]
@avibrazil myetherwallet has a "Mnemonic Phrase" option. If you generate a phrase on iancoleman then put it in there, the resulting addresses match.

**Comment by avibrazil** [comment]
@thinkloop indeed, I had to download the wallet to find and test this function. It works well.
It is not very well advertised in their website.

Thank you for letting me know about it. It became my default HD wallet for ETH.

**Comment by avibrazil** [comment]
Coinomi closed my ticket saying:

> Ledger and Coinomi use the same derivation path for eth, M/44H/60H/0H
> Jaxx, exodus, and many others use M/44H/60H/0H/0

This doesn't solve the problem cause you can't make Coinomi work the way "many others" work. But explains.

**Comment by tkalfigo** [comment]
I came across this thread because Exodus (desktop) was not showing my ETH funds received in Coinomi (android).

The compromise in the end is that in Exodus I cannot see all ETH funds, but in Coinomi I added another ETH wallet and manually set the deriv.path to the one used by Exodus. So in Coinomi I can see all ETH funds but they are spread across 2 wallets. 

See `3 - Set the derivation path` at https://coinomi.freshdesk.com/support/solutions/articles/29000016813-claiming-your-forked-coins on how to manually set the path for specific coin.

Is there a reason that Jaxx & Exodus use path `M/44H/60H/0H/0` while Ledger & Coinomi use `M/44H/60H/0H`? How come both are correct?
